Blockzilla/UIComponents/UIConstants.swift

```diff
@@ -351,6 +351,21 @@ struct UIConstants {
                 value: "Search in",
                 comment: "Translate only 'Search in'. App name will is rendered in a different label.")
 
+        public static let widgetOnboardingCardTitle = NSLocalizedString(
+                "WidgetOnboardingCard.Title",
+                value: "Browsing history cleared! 🎉",
+                comment: "Title shown on card view explaining the app has a widget option")
+
+        public static let widgetOnboardingCardSubtitle = NSLocalizedString(
+                "WidgetOnboardingCard.Subtitle",
+                value: "We’ll leave you to your private browsing, but get a quicker start next time with the Focus widget on your Home screen.",
```

Comment:
Use a placeable for the app name (and this kind of proves that using "Firefox" in the widget was not correct).

Reply:
Updated